File: Study/资料/02-Code/003_Java基础/99-视频配套代码/CHP07/src/Student.java
public class Student 
{
	//字段
	public String stuNO;  //学号
	public String stuName;  //姓名
	public String stuSex; //性别
	//方法
	public void sayHi()  //自我介绍
	{
		System.out.println("大家好");
		System.out.println("我的学号是:"+this.stuNO);
		System.out.println("我的姓名是:" + this.stuName);
		System.out.println("我的性别是:" + this.stuSex);
	}
	
}

File: Study/资料/02-Code/006_WinForm技术应用/99-视频配套源代码/Solution6/WindowsFormsApplication5/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WindowsFormsApplication5
{
    [Serializable]
    class Student
    {
        public string StuNum { get; set; } //学号
        public string StuName { get; set; } //姓名
        public int StuAge { get; set; } //年龄
    }
}
